doc/api/Renter.md

```diff
@@ -211,6 +212,32 @@ lists the status of all files.
 }
 ```
 
+#### /renter/prices [GET]
+
+lists the estimated prices of performing various network operations.
+
+###### JSON Response
+```javascript
+{
+      // The estimated cost of downloading one terabyte of data from the
+	  // network.
+      "downloadterabyte": "1234", // hastings
+
+	  // The estimated cost of forming a set of contracts on the network. This
+	  // cost also applies to the estimated cost of renewing the renter's set of
+	  // contracts.
+      "formcontracts": "1234", // hastings
```

Comment:
they will have to account for that themselves, or we can extend the api to take in a proposed set of parameters.